2020/day2/solution.py

```diff
@@ -0,0 +1,28 @@
+import re
+password_re = re.compile(r"(\d+)-(\d+) (\w): (\w+)")
+
+def parsePassword(line):
+    match = password_re.match(line)
+    return (int(match.group(1)), int(match.group(2)), match.group(3), match.group(4))
+
+def isValidPartOne(minCount, maxCount, character, password):
+    count = 0
+    count = sum(1 if x == character else 0 for x in password)
```

Comment:
```suggestion
    count = password.count(x)
```
You could combine this with roy's suggestion to make this function a one-liner!

FWIW, It looks like there's other opportunities to use count in this PR as well.